src/modules/commander/PreflightCheck.cpp

```diff
@@ -113,7 +113,7 @@ static bool magnometerCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &sta
 
 		if (!mag_valid) {
 			if (report_fail) {
-				mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: MAG #%u invalid", instance);
+				mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Mag #%u invalid", instance);
```

Comment:
What does "Mag invalid" mean?

Reply:
Changed to `Preflight Fail: no valid Data for Mag`. Is that better?

Reply:
Much better! Can you do it for accel/gyro too?